util/syspolicy/policy_keys_test.go

```diff
@@ -14,11 +14,12 @@ import (
 	"strconv"
 	"testing"
 
+	"tailscale.com/util/syspolicy/pkey"
 	"tailscale.com/util/syspolicy/setting"
 )
 
 func TestKnownKeysRegistered(t *testing.T) {
-	keyConsts, err := listStringConsts[Key]("policy_keys.go")
+	keyConsts, err := listStringConsts[pkey.Key]("policy_keys.go")
```

Comment:
The filename where the keys are defined should also be updated.
```suggestion
	keyConsts, err := listStringConsts[pkey.Key]("pkey/pkey.go")
```

Otherwise, the test will always pass but won't really check anything, since `policy_keys.go` no longer defines any keys.
I guess I should have defensively checked that `len(keyConsts)` is non-zero.

Reply:
(it would fail if we moved or renamed `policy_keys.go` without updating the path here. But the file is still there, it just no longer has the same const block as before).

Reply:
Done. And added the len() == 0 check too and verified it fails.